### ISSUE TITLE
Wide ANSis are being line-wrapped

### ISSUE THREAD

**Issue by epitron** [ansilove/ansilove#19]
Thanks for the excellent ansi tool!

I just wanted to mention that I've found a small issue with extra-wide ANSIs, which specify their width/height in the SAUCE TInfo1/2 fields. ansilove looks like it's ignoring these fields and treating it as 80-column ANSI.

(The SAUCE specification for FileType 1's TInfo fields can be found here: http://www.acid.org/info/sauce/sauce.htm#FileType)

Here's an example of an extra-wide ANSI:
https://pc.textmod.es/pack/download/blocktronics-6710/misfit-astromech.ans

The latest release (3.0.9) renders it like this:
<img src="https://0x0.st/stQ3.png" width=300>

While the master branch renders it like this:
<img src="https://0x0.st/stQx.png" width=300>

(Aside: do you think it would be a good idea to allow the `-c` option to work on ANSIs as well as BINs? You know, in case there are some wide ANSIs without proper SAUCE records.)

**Comment by fcambus** [comment]
Yes, I think it might be a good idea and I'm considering allowing using the -c option for the ANSI loader as well. My priority for now is stabilizing libansilove, then adding new features.

**Comment by Kirkman** [comment]
Just want to +1 this.

**Comment by Kirkman** [comment]
@epitron, if you want a temporary workaround for this, try the changes I made in my commit here: https://github.com/Kirkman/libansilove/commit/aaccb6e1342d104bcd61a9f01e5d58fda8969432

Of course, this is not the right solution, because there should be a different default for BINs. But if you specify the correct columns on the command line, this change will cause ansilove to properly render your ANSI above and other problematic ones in the latest Blocktronics pack.

**Comment by fcambus** [comment]
I pushed some commits in both ansilove and libansilove to address this concern, and it's now possible to use the -c switch for ANSI files as well.

The example ANSI can now be rendered correctly with: ansilove -c150 misfit-astromech.ans

I'm not too keen about using values from SAUCE files yet, I will evaluate this in the future.

I also have a local branch to support the 24-bit color sequences from PabloDraw.

**Comment by Kirkman** [comment]
I tried the new code this morning, and it seems to be working well.

**Comment by Kirkman** [comment]
FWIW, there was a similar problem with wide Tundra files (such as `tcf-shocktronics.tnd` in the Blockronics 6710 pack). I made a [pull request](https://github.com/ansilove/libansilove/pull/3) for libansilove which would allow users to request a specific column width when rendering Tundra files, same as what you did for BIN.

But there was a second problem with determining the height of Tundra files, and I proposed a change to address it in that same pull request.
